### PR TITLE
Uat fixes sheet 1

### PR DESCRIPTION
**Pull request by padmashreeshetty123**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://uat-fixes-sheet-1--world-bank--aemsites.hlx.page/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -112,7 +112,7 @@ header .nav-menu-overlay .nav-menu {
 /* Dropdown Text */
 .language-toggle {
   padding-bottom: 14px;
-  color: var(--text-on-light-link-primary-enabled);
+  color: var(--primary-blue-80);
```
Comment (thomsebastin) [inline blocks/header/header.css:115]: Please check my previous PR.

File: blocks/header/header.css [hunk 2]
```diff
@@ -121,7 +121,7 @@ header .nav-menu-overlay .nav-menu {
 }
 
 .language-text {
-  color: var(--text-on-light-link-primary-enabled);
+  color: var(--primary-blue-80);
```
Comment (thomsebastin) [inline blocks/header/header.css:124]: Don't change this and don't use old variables

File: blocks/header/header.css [hunk 3]
```diff
@@ -162,22 +162,26 @@ header .nav-menu-overlay .nav-menu a {
...
-  text-decoration: underline;
+  color: var(--primary-blue-80);
   display: block;
   font-size: 14px;
   line-height: 17px;
+  text-decoration: none;
+}
+
+header .nav-menu-overlay .nav-menu a:visited {
+  color: var(--neutrals-40);
```
Comment (thomsebastin) [inline blocks/header/header.css:173]: Use new variable values.

**Review by mkbansal1 — APPROVED** [review]

File: blocks/header/header.css [hunk 4]
```diff
@@ -189,6 +193,7 @@ header .nav-menu-overlay .nav-menu-column.left ul a {
 
 header .nav-menu-overlay .nav-menu .submenu a:hover {
   color: var(--color-white);
+  border-bottom:1px solid var(--color-white)
```
Comment (thomsebastin) [inline blocks/header/header.css:196]: Use variable --white

File: blocks/header/header.css [hunk 5]
```diff
@@ -732,6 +738,20 @@ header nav .nav-tools {
   font-weight: normal;
 }
 
+
+.section.nav-tools .default-content-wrapper .icon {
+height: 44px;
```
Comment (thomsebastin) [inline blocks/header/header.css:743]: Indent

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Review by thomsebastin — APPROVED** [review]
Still see minor indent issues.

**Pull request merged**